airbyte-integrations/connectors/source-salesforce/source_salesforce/source.py

```diff
@@ -35,6 +35,25 @@ def check_connection(self, logger: AirbyteLogger, config: Mapping[str, Any]) ->
                 logger.warn(f"API Call limit is exceeded. Error message: '{error_data.get('message')}'")
                 return False, "API Call limit is exceeded"
 
+    @staticmethod
+    def get_user_excluded_fields(config: Mapping[str, Any], stream_name: str) -> List[str]:
+        """Retrieve user-excluded fields (specified as Stream.Field) for this stream from config"""
+        excluded_fields = []
+        if config.get("exclude_fields") and stream_name is not None:
+            for f in config["exclude_fields"]:
+                if "." in f:
+                    if f.split(".")[0] == stream_name:
+                        excluded_fields.append(f.split(".")[1])
```

Comment:
```suggestion
        if config.get("exclude_fields") and stream_name is not None:
            for stream_field in config["exclude_fields"]:
                 try:
                     exclude_stream_name, exclude_field_name = stream_field.split(".")
                 exception ValueError as e:
                     raise "Stream.Field specified is not correct"
                 if exclude_stream_name == stream_name:
                      excluded_fields.append(exclude_field_name)
                      
```

What do you think about this?

Reply:
```suggestion
        if config.get("exclude_fields") and stream_name is not None:
            for stream_field in config["exclude_fields"]:
                 try:
                     exclude_stream_name, exclude_field_name = stream_field.split(".")
                 exception ValueError as e:
                     raise f"Format for excluded field [{stream_field}] is incorrect. Expected format is Stream.Field"
                 if exclude_stream_name == stream_name:
                      excluded_fields.append(exclude_field_name)
```
Variable name change makes sense.
I think your recommendation of the split into two variables is easier to read.
try...catch makes sense but if we are going to raise an exception on formatting error, I think we should include the input details in addition to the error description to make it easier for the user to resolve.

Reply:
Of course! much better.